README.md

```diff
@@ -1,7 +1,13 @@
 # Hardware Library
 The hardware library provides an abstraction layer for hardware interfaces across platforms such as:
+* linux
 * nos-linux
 
+Note that the hardware library does not include `cfe.h` or `osapi.h`.
+When using with cFS these includes must be in the `cpuX_device_cfg.h` file.
```

Comment:
This is the cavate to this update.  If these updates migrate elsewhere be sure to do this!